.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,21 @@
+name: spellcheck
+
+on:
+  push:
+    branches:
+      - master
+      - devel
+  pull_request:
+    branches:
+      - master
+      - devel
+
+jobs:
+  spellcheck:
+    name: runner / spellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code.
+      uses: actions/checkout@v3
+    - name: Run PySpelling as a GitHub action
+      uses: sbates130272/spellcheck-github-actions@v0.7.0
```

Comment:
lets change this workflow with the new github action https://github.com/marketplace/actions/github-spellcheck-action